CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Handle see all/see less button clicks for pending overdue list section
 - Fix views flashing in the overdue list
 - When overdue appointments are loaded or are loading, render appropriate views
+- Handle the rendering of pending list footer item
```

Comment:
"Handle the rendering of pending list footer item" sounds like a really generic change. Can we instead change the changelog and title to something like "Don't show pending overdue list footer if there are no pending overdue appointments" instead?